File: python/bubble_sort.py
num_list = [3, 6, 5, 9, 1, 4, 2, 8, 7]

for i in range(0, len(num_list)-1):
    for j in range(0, len(num_list)-1-i):
        if num_list[j] > num_list[j+1]:
            temp = num_list[j]
            num_list[j] = num_list[j+1]
            num_list[j+1] = temp

print(num_list)

File: javascript/bubbleSort.js
const numList = [3, 6, 5, 9, 1, 4, 2, 8, 7]

let temp = 0
for (let i=0 ; i<numList.length-1 ; i++){
  for (let j=0 ; j<numList.length-1-i ; j++){
    if(numList[j] > numList[j+1]){
      temp = numList[j]
      numList[j] = numList[j+1]
      numList[j+1] = temp
    }
  }
}

console.log(numList)